### DISCUSSION TITLE
Feedback for “HuggingFace”

### DISCUSSION THREAD

**Discussion by toledoroy** [LibreChat-AI/docs#145] [Q&A]
The page: https://www.librechat.ai/docs/remote/huggingface 
Contains a broken link to https://huggingface.co/spaces/LibreChat/template

**Comment by fuegovic** [comment]
you mean this? 
![image](https://github.com/user-attachments/assets/8c984e4a-8211-428a-b905-82f3d587acd5)

This is normal, the template is just a template. 
It needs to be duplicated and configured to properly run
